web/pdf_viewer.js

```diff
@@ -1099,10 +1114,27 @@ class PDFViewer {
       newScale * PixelsPerInch.PDF_TO_CSS_UNITS
     );
 
-    const updateArgs = { scale: newScale };
+    const mustPostponeDrawing = drawingDelay >= 0 && drawingDelay < 1000;
+    const updateArgs = {
+      scale: newScale,
+    };
+    if (mustPostponeDrawing) {
+      updateArgs.postponeDrawingDelay = drawingDelay;
```

Comment:
Again, can't we just call the parameter `drawingDelay` consistently everywhere?
```suggestion
      updateArgs.drawingDelay = drawingDelay;
```